README.md

```diff
@@ -77,20 +76,12 @@ In this repository [distgen](https://github.com/devexp-db/distgen/) is used for
 Usage
 ---------------------------------
 
-For information about usage of Dockerfile for PostgreSQL 9.6,
-see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/9.6).
-
 For information about usage of Dockerfile for PostgreSQL 10,
 see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/10).
 
 For information about usage of Dockerfile for PostgreSQL 12,
 see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/12).
 
-For versions which are not supported anymore:
-[PostgreSQL 9.2](https://github.com/sclorg/postgresql-container/blob/f213e5d0/9.2)
```

Comment:
Is this removal deliberate? It actually makes sense to keep such references for anybody who needs some EOL version and does not care it is deprecated already.